bundle/manifests/search.open-cluster-management.io_searches.yaml

```diff
@@ -0,0 +1,349 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
@xrajesh Some of these files get updated automatically. What's the best way to ensure that these stay in sync with code changes in the operator peroject?

Reply:
@xrajesh If the operator is running with different version of CRD (1) Bundle install should fail if the versions mismatch - , when we apply the CR for the operator (2) If there is field changes that should be tested in our e2e for all the options.